--- meson.build.orig	2026-08-19 13:44:40.528211404 +0800
+++ meson.build	2026-08-19 13:44:40.764211404 +0800
@@ -4,7 +4,7 @@
     'c',
     'cpp',
     'cython',
-    version: run_command(['generate_version.py', '--print'], check: true).stdout().strip(),
+    version: '3.0.5',
     license: 'BSD-3',
     meson_version: '>=1.2.1',
     default_options: ['buildtype=release', 'c_std=c11', 'warning_level=2'],
