--- a/pandas/meson.build
+++ b/pandas/meson.build
@@ -1,22 +1,2 @@
-incdir_numpy = run_command(
-    py,
-    [
-        '-c',
-        '''
-import os
-import numpy as np
-try:
-    # Check if include directory is inside the pandas dir
-    # e.g. a venv created inside the pandas dir
-    # If so, convert it to a relative path
-    incdir = os.path.relpath(np.get_include())
-except Exception:
-    incdir = np.get_include()
-print(incdir)
-     ''',
-    ],
-    check: true,
-).stdout().strip()
-
-inc_np = include_directories(incdir_numpy)
+inc_np = include_directories('/data/data/com.termux/files/usr/lib/python3.14/site-packages/numpy/_core/include')
 inc_pd = include_directories('_libs/include')